Cargo.toml

```diff
@@ -59,8 +59,8 @@ test-electrum = ["electrum"]
 test-md-docs = ["electrum"]
 
 [dev-dependencies]
-bdk-testutils = "^0.3"
-bdk-testutils-macros = "^0.3"
+bdk-testutils = { path = "./testutils"}
```

Comment:
I'm thinking that maybe we should always use local dependencies in our development branches and only replace them with version number right before the release. It would need to change the way we do releases though, because we would have to merge the release branch before we do the bump. Something like:

```
master -------------------------------------------- bump dev ------ ...
           \                                  /
release     --- bump rc.1 --- fix --- fix --- ---- publish deps --- bump release version
```

Don't know if this would be better, just thinking out loud

Reply:
Yes I agree,  I ran into the same issue when testing locally. We should also add a step to the DEVELOPMENT_CYCLE.md so we don't forget.. but I'll do that in a separate PR.